lib/library.js

```diff
@@ -51,8 +53,7 @@ function Library (libfile, funcs, lib) {
       , info = funcs[func]
 
     if (fptr.isNull()) {
-      throw new Error('Library: "' + libfile
```

Comment:
If you tack `this.path = path` in the DynamicLibrary constructor then we can still know the name of the dylib and print it out here (`dl.path`).